### ISSUE TITLE
1.16.1 kicked from server, does not work

### ISSUE THREAD

**Issue by Saharo4eks** [AoElite/AutoReconnect#10]
Proxy: BungeeCord#1515
Server: PaperSpigot#10 [1.16.1]

The plugin does not have any functionality, when restarting the main server, I just kick with BungeeCord. There are no errors, the plugin is enabled normally.

Videos: https://www.youtube.com/watch?v=pBz8iOkJKYM&feature=youtu.be

**Comment by PseudoResonance** [comment]
I couldn't see the bottom of your config file, but did you make sure that your kick message is caught by the plugin? At the bottom of the config, there is a section that looks like this:

As you can see, the default kick message is "Server closed". The plugin only attempts to reconnect you if it finds that exact message. You can also enable regex and use a regex pattern to check whether or not it should try to reconnect. You can test the regex out on a website such as this one: https://regexr.com/
```yml
# This block is necessary for the plugin to differentiate whether a kick happened because of a shutdown or not.
shutdown:
  # When regex is set to false, the kick message must be equal to the following text if the player should be reconnected to that server. (Formatting codes will be ignored though.)
  # When regex is set to true, the regex must match the kick message if the player should be reconnected to that server.
  # Leaving the text empty will lead the plugin to always reconnect the players, no matter what the actual kick-reason was.

  # You can still use the "ServerReconnectEvent" as a plugin developer to decide on your own whether the user should be reconnected automatically
  # or not.

  text: "Server closed"
  regex: false
```

**Comment by sorienrot** [comment]
```
shutdown:
  # When regex is set to false, the kick message must be equal to the following text if the player should be reconnected to that server. (Formatting codes will be ignored though.)
  # When regex is set to true, the regex must match the kick message if the player should be reconnected to that server.
  # Leaving the text empty will lead the plugin to always reconnect the players, no matter what the actual kick-reason was.

  # You can still use the "ServerReconnectEvent" as a plugin developer to decide on your own whether the user should be reconnected automatically
  # or not.

  text: "Server closed"
  regex: 
```
 **Leaving the text empty will lead the plugin to always reconnect the players**

I've put it this way and it doesn't try to reconnect the players, it automatically disconnects them. Reconnection only works if I remove protocolize-plugin.jar plugin.
I have installed protocolize-plugin.jar for the limbo world, but it always disconnects, never reconnects.

**Comment by PseudoResonance** [comment]
I do not know why you have removed the `true` or `false` from regex, but your server's disconnection message is "Server closed" exactly, and it only works without Protocolize? Are you running the latest version of the plugin, version 1.5, as well as the latest version of Protocolize, version 1.6.1? And are there any logs created?

**Comment by SirOMGitsYOU** [comment]
I'm not using Protocolize, and whenever I restart the server I just get booted, freshly installed the plugin no edits been made.

**Comment by PseudoResonance** [comment]
No reply, so assuming issue is resolved.
1.16 support addressed in 302fbe01fde7ea83203e0a064b244cc0ab817a93
